flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -1262,14 +1277,8 @@ private CompletableFuture<Void> removeJob(JobID jobId, CleanupJobState cleanupJo
         }
     }
 
-    private void markJobAsClean(JobID jobId) {
-        try {
-            jobResultStore.markResultAsClean(jobId);
-            log.debug(
-                    "Cleanup for the job '{}' has finished. Job has been marked as clean.", jobId);
-        } catch (IOException e) {
-            log.warn("Could not properly mark job {} result as clean.", jobId, e);
-        }
+    private CompletableFuture<Void> markJobAsCleanAsync(JobID jobId) {
```

Comment:
You would rather have a `thenRun` in this method to trigger the debug log message.

Reply:
Fixed.